tests/integration/test_schema.py

```diff
@@ -18,11 +17,16 @@ def test_table_schema(c, df):
     c.sql('USE SCHEMA "foo"')
     assert_eq(original_df, c.sql("SELECT * FROM root.df"))
 
-    c.sql("CREATE TABLE bar AS TABLE root.df")
+    # TODO: https://github.com/dask-contrib/dask-sql/issues/771
+    # For now, we create table bar another (equivalent) way
+    # c.sql("CREATE TABLE bar AS TABLE root.df")
+    c.sql("CREATE TABLE bar AS (SELECT * FROM root.df)")
     assert_eq(original_df, c.sql("SELECT * FROM bar"))
 
     with pytest.raises(KeyError):
-        c.sql("CREATE TABLE other.bar AS TABLE df")
+        # TODO: https://github.com/dask-contrib/dask-sql/issues/771
+        # c.sql("CREATE TABLE other.bar AS TABLE df")
+        c.sql("CREATE TABLE other.bar AS (SELECT * FROM df)")
```

Comment:
This currently errors because there is no `foo.df`, and the nested statement is parsed before we look at `other.bar`. I could fix this by changing it to `(SELECT * FROM root.df)`